model/model.go

```diff
@@ -91,6 +91,19 @@ type ExposedPort struct {
 	ExposedPort int `json:"exposedPort" yaml:"exposedPort"`
 }
 
+type ExecAction struct {
+	Command []string `json:"command,omitempty" yaml:"command,omitempty"`
```

Comment:
I wonder why it's the only array with `omitempty` specified. Probably it's unneeded here.
So, would be cool if you could test without it.

Reply:
I will check

Reply:
Checked, you are right `omitempty` not needed going to remove it